app/src/main/java/net/kwatts/powtools/components/LockableScrollView.java

```diff
@@ -0,0 +1,62 @@
+package net.kwatts.powtools.components;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+import android.widget.ScrollView;
+
+/**
+ * Copied from https://stackoverflow.com/a/5763815/247325
+ */
+public class LockableScrollView extends ScrollView {
+
+    // true if we can scroll (not locked)
+    // false if we cannot scroll (locked)
+    private boolean mScrollable = true;
+
+    public LockableScrollView(Context context) {
+        super(context);
+    }
+
+    public LockableScrollView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    public LockableScrollView(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+    }
+
+    public LockableScrollView(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
+        super(context, attrs, defStyleAttr, defStyleRes);
+    }
+
+    public void setScrollingEnabled(boolean enabled) {
+        mScrollable = enabled;
+    }
+
+    public boolean isScrollable() {
+        return mScrollable;
+    }
+
+    @Override
+    public boolean onTouchEvent(MotionEvent ev) {
+        switch (ev.getAction()) {
+            case MotionEvent.ACTION_DOWN:
+                // if we can scroll pass the event to the superclass
+                if (mScrollable) return super.onTouchEvent(ev);
+                // only continue to handle the touch event if scrolling enabled
+                return mScrollable; // mScrollable is always false at this point
+            default:
+                return super.onTouchEvent(ev);
+        }
+    }
+
+    @Override
+    public boolean onInterceptTouchEvent(MotionEvent ev) {
+        // Don't do anything with intercepted touch events if
+        // we are not scrollable
+        if (!mScrollable) return false;
+        else return super.onInterceptTouchEvent(ev);
```

Comment:
Curly braces would make this more consistent with the codebase and easier to read.